src/pages/WishlistCard.js

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import "../App.css";
+import { useUserDetail } from "../context/userdetail-context";
+function WishlistCard( { item }) {
+  const { userDetail , setUserDetail } = useUserDetail();
+  const { wishlist } = userDetail;
+  const { imgurl ,title ,rating ,price  } = item;
+  function removeWishHandler (prod) {
+      const RevomveWishList = wishlist.filter((item) => item._id !== prod._id );
```

Comment:
A typo here might cause an issue at later point

Reply:
Thank you so much Sadath for pointing out